release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,932 @@
+
+[[ocp-39-cloudforms]]
+==== CloudForms 4.6 Container Management
+
+* {product-title} template provisioning
+* Offline OpenScapScans
+* Alert management: You can choose Prometheus (currently in Technology Preview) and use it in CloudForms.
+* Reporting enhancements
+* Provider updates
+* Chargeback enhancements
+* UX enhancememts
+
+[[ocp-39-crio]]
+==== CRI-O v1.9 (Technology Preview)
+
+CRI-O is a feature currently in xref:ocp-39-technology-preview[Technology
+Preview] and not for production workloads.
+
+CRI-O is a lightweight, native Kubernetes container runtime interface. By
+design, it provides only the runtime capabilities needed by the kubelet. CRI-O is
+designed to be part of Kubernetes and evolve in lock-step with the platform.
+
+CRI-O brings:
+
+* A minimal and secure architecture.
+* Excellent scale and performance.
+* The ability to run any Open Container Initiative (OCI) or docker image.
+* Familiar operational tooling and commands.
+
+image::crio-3-7.png[CRI-O]
+
+To install and run CRI-O alongside `docker`, set the following in the
+`[OSEv3:vars]` section
+xref:../install_config/install/advanced_install.adoc#configuring-ansible[Ansible inventory file] during cluster installation:
+
+----
+openshift_use_crio=true
+----
+
+This setting pulls the *openshift3/cri-o* system container image from the
+link:https://access.redhat.com/containers[Red Hat Registry] by default. If you
+want to use an alternative CRI-O system container image from another registry,
+you can also override the default using the following variable:
+
+----
+openshift_crio_systemcontainer_image_override=<registry>/<repo>/<image>:<tag>
+----
+
+[NOTE]
+====
+The `atomic-openshift-node` service must be RPM- or system container-based when
+using CRI-O; it cannot be `docker` container-based. The installer protects again
+using CRI-O with `docker` container nodes and will halt installation if
+detected.
+====
+
+When CRI-O use is enabled, it is installed alongside `docker`, which currently
+is required to perform build and push operations to the reigstry. Over time,
+temporary `docker` builds can accumulate on nodes. You can optionally set the
+following to enable garbage collection, which adds a daemonset to clean out the
+builds:
+
+----
+openshift_crio_enable_docker_gc=true
+----
+
+When enabled, it will run garbage collection on all nodes by default. You can
+also limit the running of the daemonset on specific nodes by setting the
+following:
+
+----
+openshift_crio_docker_gc_node_selector={'runtime': 'cri-o'}
+----
+
+For example, the above would ensure it is only run on nodes with the `runtime:
+cri-o` label. This can be helpful if you are running CRI-O only on
+xref:../install_config/build_defaults_overrides.adoc#ansible-setting-global-build-defaults[some
+nodes], and others are only running `docker`.
+
+See the link:http://cri-o.io/[upstream documentation] for more information on
+CRI-O.
+
+[[ocp-39-buildha]]
+==== Buildha
```

Comment:
typo, should be "buildah" (same for anchor text above)

Reply:
I am a little confused why this is in an openshift release note?

Reply:
@bparees - could you confirm it shouldn't be here? cc: @ahardin-rh for https://bugzilla.redhat.com/show_bug.cgi?id=1557246

Reply:
I also wasn't sure why it was here, so if @derekwaynecarr doesn't think it should be either, i'm all for removing it.  There's nothing about it that's actually a ocp 3.9 feature/tool.